lms/envs/common.py

```diff
@@ -948,6 +948,7 @@ def _add_microsite_dirs_to_default_template_engine(settings):
 DEFAULT_FEEDBACK_EMAIL = 'feedback@example.com'
 SERVER_EMAIL = 'devops@example.com'
 TECH_SUPPORT_EMAIL = 'technical@example.com'
+TECH_SUPPORT_LINK = 'http://pearsonadvance.echelp.org/'
```

Comment:
Https? @cocococosti

Reply:
I thought the same thing but the url is http in the ticket @Squirrel18